tests/components/neato/test_init.py

```diff
@@ -20,27 +20,53 @@
     CONF_VENDOR: VENDOR_NEATO,
 }
 
+DIFFERENT_CONFIG = {
+    CONF_USERNAME: "anotherUsername",
+    CONF_PASSWORD: "anotherPassword",
+    CONF_VENDOR: VENDOR_VORWERK,
+}
+
 INVALID_CONFIG = {
     CONF_USERNAME: USERNAME,
     CONF_PASSWORD: PASSWORD,
     CONF_VENDOR: VENDOR_INVALID,
 }
 
 
-@pytest.fixture(name="account")
-def mock_controller_login():
+@pytest.fixture(name="configflow")
```

Comment:
Please separate words with underscore in names.

`"config_flow"`

Reply:
Done